tests/test_cohort_joiner.py

```diff
@@ -41,7 +42,16 @@ class TestWriteDataframe:
     def test_write_csv(self, tmp_path, dataframe):
         csv_path = tmp_path / "input.csv"
         cohort_joiner.write_dataframe(dataframe, csv_path)
-        assert csv_path.exists()
+        # When writing a dataframe to a CSV, it's easy to write the dataframe's index,
```

Comment:
Oh thanks, this is nice! @ccunningham101 and I ran into this problem while thinking about the decile chart action